PocketKit/Sources/PocketKit/Settings/AccountViewModel.swift

```diff
@@ -6,10 +6,15 @@ import SharedPocketKit
 import SwiftUI
 
 class AccountViewModel: ObservableObject {
+    static let ToggleAppBadgeKey = "AccountViewModel.ToggleAppBadge"
```

Comment:
style nitpick: lowercase variable name? `toggleAppBadgeKey`

Reply:
got it. I often do static global variables be UpperCaseCamel.  I'll change it though

Reply:
no worries, im actually not sure what the standard is on this team... so could be just my preference. Not sure if we should standardize this or not